data_inconsistencies.py

```diff
@@ -32,6 +33,35 @@ def get_inconsistencies():
     return incons
 
 
+def inconsistencies_metric(file_name):
```

Comment:
Rename the function to `print_statistics` (inconstincies is implied by the file name)